lib/kino_explorer/data_transform_cell.ex

```diff
@@ -910,17 +910,15 @@ defmodule KinoExplorer.DataTransformCell do
   defp valid_data(%DataFrame{}), do: true
 
   defp valid_data(data) do
-    with true <- implements?(Table.Reader, data),
-         {_, %{columns: [_ | _] = columns}, _} <- Table.Reader.init(data),
-         true <- Enum.all?(columns, &implements?(String.Chars, &1)) do
-      true
-    else
-      _ -> false
+    try do
+      data = DataFrame.new(data)
+      {data, false}
+    rescue
```

Comment:
We use the dataframe only to get dtypes, so instead of carrying the whole dataframe around, let's just compute the dtypes here.

Also, let's keep the table reader check, so that we don't try to build dataframe for every variable, it's probably cheaper.